pom.xml

```diff
@@ -72,5 +72,15 @@
             <artifactId>sapphire</artifactId>
             <version>1.0.0-R0.1-SNAPSHOT</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.xerial</groupId>
+            <artifactId>sqlite-jdbc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.mysql</groupId>
+            <artifactId>mysql-connector-j</artifactId>
+            <version>${mysql.version}</version>
+        </dependency>
```

Comment:
I went ahead and removed the mariadb dependency. I don't think it's necessary with the mysql-connector (which I also updated to use the version provided by the parent project).

Reply:
Do you think I should remove the MariaDBConnection in generel or keep it for anything in futute?
![image](https://github.com/user-attachments/assets/31fce95b-dd3a-42de-8d1d-c64b3c5ad60d)

Reply:
MariaDB and MySQL *should* be able to operate off the same MySQL driver, I believe. So it's fine to stay. Our other plugins don't have explicit support for maria db though

Reply:
Alrighty